benchmarks/src/lib/constants/benchmark-tests.ts

```diff
@@ -1,403 +1,640 @@
-import { MathOperator, Method } from '@joshdb/provider';
+import { MathOperator, Method, Payload } from '@joshdb/provider';
 import type { Benchmark } from '../Benchmark';
 
 export const BENCHMARK_TESTS: Benchmark.Test[] = [
   {
     name: Method.AutoKey,
 
     run: async ({ josh }) => {
-      await josh.autoKey();
+      await josh.autoKey({ method: Method.AutoKey, errors: [] });
```

Comment:
```suggestion
      await josh[Method.AutoKey]({ method: Method.AutoKey, errors: [] });
```
This should be applied everywhere